.gitignore

```diff
@@ -25,6 +25,7 @@ yarn-error.log*
 .directory
 
 /functions/lib
+/functions/node_modules
```

Comment:
What is the need? `node_modules` are ignored recursively, see in the beginning of this file.